### PR TITLE
Feat/tags only when request

### PR DESCRIPTION
**Pull request by acien101**
Tags are not executed unless a tag is specifically requested.

### CODE REVIEW

File: full.yml [hunk 1]
```diff
@@ -555,17 +555,18 @@
...
     docker_command: |
       /usr/bin/docker run --rm --name "{{ service_name }}" \
-      -v /var/run/docker.sock:/var/run/docker.sock \
       -v "{{ drone_directory.data }}":/data \
       -p "{{ drone_internal_web_port }}:80" \
       -e "USER_UID=1000" -e "USER_GID=1000" \
       -e "DRONE_GITEA_SERVER=https://{{ gitea_domain }}" \
+      -e "DRONE_GITEA_CLIENT_ID={{ gitea_oauth_client_id }}" \
+      -e "DRONE_GITEA_CLIENT_SECRET={{ gitea_oauth_client_secret }}" \
+      -e "DRONE_RPC_SECRET={{ drone_rpc_scret }}" \
```
Comment (m0wer) [inline full.yml]: This shouldn't be in this PR.

File: full.yml [hunk 2]
```diff
@@ -601,7 +602,30 @@
...
+      -p "{{ drone_docker_runner_internal_web_port }}:3000" \
+      -e "DRONE_RPC_PROTO=https" \
+      -e "DRONE_RPC_HOST={{ drone_domain }}" \
+      -e "DRONE_RPC_SECRET={{ drone_rpc_scret }}"" \
+      -e "DRONE_RUNNER_CAPACITY=2" \
+      -e "DRONE_RUNNER_NAME={{ drone_runner_name }}"" \
+      "{{ docker_image }}"
+  roles:
+    - role: generic_docker_systemd
+  tags: [ never, drone ]
```
Comment (m0wer) [inline full.yml]: Same.

File: group_vars/all.yml [hunk 3]
```diff
@@ -151,6 +151,16 @@ drone_domain: "drone.{{ base_domain }}"
...
+gitea_oauth_client_id: "changeme"
+gitea_oauth_client_secret: "changeme"
+drone_rpc_scret: "changeme"
+
+## Drone docker runner
+
+drone_docker_runner_docker_image: drone/agent:1
+drone_docker_runner_internal_web_port: 3000
+drone_runner_name: "drone-runner.{{ base_domain }}"
+
```
Comment (m0wer) [inline group_vars/all.yml]: Same.

**Comment by acien101** [comment]
There you have

**Comment by m0wer** [comment]
Ok perfect. Thanks :-)

**Pull request merged**